web/src/input_pill.ts

```diff
@@ -12,6 +12,7 @@ import * as ui_util from "./ui_util";
 // See https://zulip.readthedocs.io/en/latest/subsystems/input-pills.html
 
 export type InputPillItem<T> = {
+    id: string;
```

Comment:
I think would be better if this is `group_id` since there is ongoing work for showing popover for users as well and that would add `user_id` field as well.